### PR TITLE
fabric-crashlytics-deployer for android 1.0.0

### PR DESCRIPTION
**Pull request by manuelgon47**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by doomsayer13** [comment]
Hi @manuelgon47 
Thank you for creating a PR, we will check it ASAP.

**Comment by manuelgon47** [comment]
Hi @doomsayer13 
I forgot to include the help info to use the step. To check how it works you can include the following script in your bitrise.yml:
`    - git::https://github.com/manuelgon47/bitrise-step-fabric-crashlytics-deployer@master:
       title: deploy fabric
       inputs:
       - fabric_api_key: "3a99caf78dd3f42382d5f05be12275dd4afd9e45"
       - fabric_build_secret: "9cd2d91e55a93b00ae8da937dc1142ac81b1a26f8e095aef3eca96f705807c63"
       - fabric_apk_path: "$BITRISE_SOURCE_DIR/app/app-release.apk"
       - fabric_android_res: "$BITRISE_SOURCE_DIR/app/src/main"
       - fabric_android_manifest: "$BITRISE_SOURCE_DIR/app/src/main/AndroidManifest.xml"
       - fabric_beta_distribution_notification: "Yes"
       - fabric_beta_distribution_list: "testGroup"
`
To publish the apk to Fabric Crashlytics Beta I'm using a external jar included in the same Github repo.

If you need more info, feel free to contact me.
Regards!

**Comment by trapacska** [comment]
Hey @manuelgon47 !

Thank you for the awesome PR. I found one little thing that would be nice if it would be changed.

It is the default value of: `fabric_apk_path`: `null`. If you use our **gradle-runner** step you will have the env `$BITRISE_APK_PATH` exported, so that could be the default value easily!

What do you think? 🙂

File: steps/fabric-crashlytics-deployer/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    summary: Set the fabric api key
+    title: 'Fabric: API Key'
+- fabric_build_secret: null
+  opts:
+    description: |
+      Set the fabric build secret
+    is_required: true
+    summary: Set the fabric build secret
+    title: 'Fabric: Build Secret'
+- fabric_apk_path: null
```
Comment (trapacska) [inline steps/fabric-crashlytics-deployer/1.0.0/step.yml]: Is it possible to use `$BITRISE_APK_PATH` as the default value?

Comment (manuelgon47) [reply steps/fabric-crashlytics-deployer/1.0.0/step.yml]: Sure!

Comment (manuelgon47) [reply steps/fabric-crashlytics-deployer/1.0.0/step.yml]: I open an other PR versioned to 1.0.1 with your change request
https://github.com/bitrise-io/bitrise-steplib/pull/1361

**Pull request closed without merging**